tests/snmp/test_snmp_phy_entity.py

```diff
@@ -428,26 +870,105 @@ def test_psu_info(duthosts, enum_rand_one_per_hwsku_hostname, snmp_physical_enti
+            "but got '{}'."
+        ).format(
+            name,
+            psu_snmp_fact['entPhysDescr']
+        )
+        assert psu_snmp_fact['entPhysContainedIn'] == CHASSIS_SUB_ID, (
+            "PSU containment mismatch. Expected 'entPhysContainedIn' to be '{}', but got '{}'."
+        ).format(
+            CHASSIS_SUB_ID,
+            psu_snmp_fact['entPhysContainedIn']
+        )
+        assert psu_snmp_fact['entPhysClass'] == PHYSICAL_CLASS_POWERSUPPLY, (
+            "PSU class mismatch. Expected class 'PHYSICAL_CLASS_POWERSUPPLY', but got '{}'."
+        ).format(
+            psu_snmp_fact['entPhysClass']
+        )
+        assert psu_snmp_fact['entPhyParentRelPos'] == position, (
+            "PSU relative position mismatch. Expected 'entPhyParentRelPos' to be '{}', "
+            "but got '{}'."
+        ).format(
+            position,
+            psu_snmp_fact['entPhyParentRelPos']
+        )
+        assert psu_snmp_fact['entPhysName'] == name, (
+            "PSU name mismatch. Expected 'entPhysName' to be '{}', but got '{}'."
+        ).format(
+            name,
+            psu_snmp_fact['entPhysName']
+        )
+        assert psu_snmp_fact['entPhysHwVer'] == '', (
+            "PSU hardware version mismatch. Expected 'entPhysHwVer' to be empty string, "
+            "but got '{}'."
+        ).format(
+            psu_snmp_fact['entPhysHwVer']
+        )
+        assert psu_snmp_fact['entPhysFwVer'] == '', (
+            "PSU firmware version mismatch. Expected 'entPhysFwVer' to be empty string, "
+            "but got '{}'."
+        ).format(
+            psu_snmp_fact['entPhysFwVer']
+        )
+        assert psu_snmp_fact['entPhysSwVer'] == '', (
+            "PSU software version mismatch. Expected 'entPhysSwVer' to be empty string, "
+            "but got '{}'."
+        ).format(
+            psu_snmp_fact['entPhysSwVer']
+        )
+        assert psu_snmp_fact['entPhysSerialNum'] == (
+            '' if is_null_str(psu_info['serial']) else psu_info['serial']
+        ), (
+            "PSU serial number mismatch. Expected '{}' but got '{}'."
+        ).format(
+            '' if is_null_str(psu_info['serial']) else psu_info['serial'],
+            psu_snmp_fact['entPhysSerialNum']
+        )
+        assert psu_snmp_fact['entPhysMfgName'] == '', (
+            "PSU manufacturer name mismatch. Expected 'entPhysMfgName' to be empty string, "
+            "but got '{}'."
+        ).format(
+            psu_snmp_fact['entPhysMfgName']
+        )
         assert psu_snmp_fact['entPhysModelName'] == '' if is_null_str(
-            psu_info['model']) else psu_info['model']
-        assert psu_snmp_fact['entPhysIsFRU'] == REPLACEABLE if psu_info['is_replaceable'] == 'True' else NOT_REPLACEABLE
+            psu_info['model']) else psu_info['model'], (
+            "PSU model name mismatch. Expected '{}' but got '{}'."
+        ).format(
+            '' if is_null_str(psu_info['model']) else psu_info['model'],
```

Comment:
For the summary, in the format, we usually print out the necessary variables, not need to do comparsion.
Print `psu_info['model']` and `psu_snmp_fact['entPhysModelName']` is enough